src/inmanta/ast/__init__.py

```diff
@@ -38,8 +38,7 @@
     from inmanta.ast.statements.define import DefineEntity, DefineImport  # noqa: F401
     from inmanta.ast.type import NamedType, Type  # noqa: F401
     from inmanta.compiler import Compiler
-    from inmanta.execute.runtime import DelayedResultVariable, ExecutionContext, Instance, ResultVariable  # noqa: F401
-    from inmanta.plugins import PluginException
+    from inmanta.execute.runtime import DelayedResultVariable, ExecutionContext, Instance, ResultVariable, Unknown  # noqa: F401
```

Comment:
Do you know why we have the noqa here?

Reply:
I added it so pep8 doesn't complain the import is not used (but the import is there to keep the symbols in place)

Reply:
But this one was already here.